go/chat/convsource.go

```diff
@@ -939,7 +874,7 @@ func (s *HybridConversationSource) GetMessagesWithRemotes(ctx context.Context,
 	}
 	if len(merges) > 0 {
 		sort.Sort(ByMsgID(merges))
-		if err = s.mergeMaybeNotify(ctx, convID, uid, merges); err != nil {
+		if err := s.mergeMaybeNotify(ctx, convID, uid, merges); err != nil {
```

Comment:
Curious why this change?

Reply:
I changed the return type of `FetchMessages` to be `storage.Error`, which failed here.

Reply:
Do you mean `GetMessagesWithRemotes`, which is where this line is?